c2cgeoportal/scripts/c2ctool.py

```diff
@@ -214,10 +214,29 @@ def get_project():
             return yaml.safe_load(f)
 
     def test_checkers(self):
-        http = httplib2.Http()
+        # Read project options for checker http session connection
+        try:
+            checker_protocol = self.project["checker_protocol"]
```

Comment:
On 2.3 we just add a `checker_url` I think that better to already use the same things, and keep the old parameters it not present...

Reply:
OK, I will try to backport the "checker_url" part from 2.3 in this PR.